src/components/Heading.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const StyledHeading = styled.div`
+  color: var(--tints-blizzard-black);
+  text-align: center;
+  padding: 4rem 1.5rem 3rem;
+  max-width: 35.1875rem;
+
+  .brow {
+    font-size: 1.125rem;
+    font-weight: 600;
+    text-transform: uppercase;
+  }
+
+  .heading {
+    font-size: 2rem;
+    font-weight: 700;
+  }
+
+  .description {
```

Comment:
I think we also need to change the font-size for 'description' to '20px'. But I'm working the font-system ticket so it may be fixed with that once I'm done with it.

Reply:
Finally got around to incorporating that. It works really well.